lib/ii.c

```diff
@@ -141,6 +141,28 @@ uint8_t ii_leader_enqueue( uint8_t address
     return 0;
 }
 
+uint8_t ii_leader_enqueue_bytes( uint8_t  address
+                               , uint8_t* data
+                               , uint8_t  tx_len
+                               , uint8_t  rx_len
+                               )
+{
+    int ix = queue_enqueue( l_qix );
+    if( ix < 0 ){ printf("queue full\n"); return 1; }
+
+    ii_q_t* q = &l_iq[ix];
+
+    if( tx_len > II_MAX_BROADCAST_LEN ){ tx_len = II_MAX_BROADCAST_LEN; }
+    if( rx_len > II_MAX_RECEIVE_LEN ){ rx_len = II_MAX_RECEIVE_LEN; }
+
+    q->address = address;
+    q->query_length = rx_len;
+    q->length = tx_len;
+    memcpy( q->data, data, tx_len );
```

Comment:
I think `q->data[0]` from the byte array would just be the command selection byte, we could pass `q->data[1]` if `tx_len > 1` though. It just seemed non-obvious to me how to interpret a binary blob as a float and unclear if that would be useful in the rx callback, but maybe so.

Reply:
I don't think either `data` or `arg` is a float. I know internally in Lua everything is technically a float, but with `ii` we're 99% of the time representing integers. Especially when it comes to a 'get' command - the arg will typically be an index ("get channel 2"), so i think it's safe to assume it's a u8.

It's probably a non-issue anyway, as most people who are going to use this interface are going to be able to track their own state (and hence know which arg is relevant in advance).

Reply:
This makes sense, I wound up passing it as the last arg to `event_raw` since it is redundant with `command` for most (all?) queries as they tend to be a single byte message. https://github.com/monome/crow/pull/367/commits/c3bea6abbca5aa76f3a401ef2d9b9fa29ec879af